ami/data/buffers/random_data_buffer.py

```diff
@@ -0,0 +1,76 @@
+from typing import Any, Self
+
+import numpy as np
+import torch
+from torch.utils.data import TensorDataset
+
+from ..step_data import DataKeys, StepData
+from .base_data_buffer import BaseDataBuffer
+
+
+class RandomDataBuffer(BaseDataBuffer):
+    """A data buffer which does not preserve data order."""
+
+    def init(self, max_len: int, key_list: list[DataKeys]) -> None:
+        """Initializes data buffer.
+
+        Args:
+            max_len (int): max length of buffer.
+            key_list (list[DataKeys]): a list of keys to save whose values to buffer.
```

Comment:
```suggestion
            max_len : max length of buffer.
            key_list : a list of keys to save whose values to buffer.
```
型記述はなくて良いです。型アノテーションによって示す事が可能となり、冗長となりました。

Reply:
42127e9: docstringから冗長な型情報を削除